EdgeMultiplay/Scripts/EdgeManager.cs

```diff
@@ -624,11 +629,50 @@ void UpdateObserverOwnership(GamePlayEvent ownershipChangeEvent)
             oldOwner.observer.observables.RemoveAt(ownershipChangeEvent.integerData[0]);
         }
 
+        void TakeOverObservable(GamePlayEvent gamePlayEvent)
```

Comment:
Can the local user reject this after consent? Is there confirmation to the server?

Reply:
So the requester can only take over the observable immediately if the current owner sat SquattingAllowed to true.

There is another way for ownership transfer by request, where the requester will send an ownership request and it's up to the owner to accept or discard the request.